src/components/graphic.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { GridContainer, Grid } from "@trussworks/react-uswds";
+
+const Graphic = () => {
+  return (
+    <GridContainer>
```

Comment:
We will most likely have a `GridContainer` in the outermost of the `Body` component or `main`, so will this be necessary? You can simply create a `Grid` and leave out the `GridContainer`.